tweepy_api/settings.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+#Variables that contains the user credentials to access Twitter API
+access_token = os.environ.get('TWITTER_ACCESS_KEY')
+access_token_secret = os.environ.get('TWITTER_SECRET_ACCESS_KEY')
+consumer_key = os.environ.get('TWITTER_CONSUMER_KEY')
+consumer_secret = os.environ.get('TWITTER_SECRET_CONSUMER_KEY')
+
+auth = tweepy.OAuthHandler(consumer_key, consumer_secret)
+auth.set_access_token(access_token, access_token_secret)
+
+api = tweepy.API(auth)
```

Comment:
this is great! This code is actually repeated in the other script you merged, so I'll remove it on my branch and we can just import that info :)